### PR TITLE
[DOCS] Write and update TextureRect, Label and NinePatchRect

### PR DESCRIPTION
**Pull request by NathanLovato**
I did 1 commit for each class. Should I squash them?

### CODE REVIEW

File: doc/classes/NinePatchRect.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="NinePatchRect" inherits="Control" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+    Scalable texture-based frame that tiles the texture's centers and sides, but keeps the corners' original size. Perfect for panels and dialog boxes.
```
Comment (Noshyaar) [inline doc/classes/NinePatchRect.xml]: Indentation issues this file.

File: doc/classes/TextureRect.xml [hunk 2]
```diff
@@ -56,29 +56,36 @@
...
 		<constant name="STRETCH_KEEP_CENTERED" value="4">
+			The texture keeps its original size and stays centered in the node's bounding rectangle.
 		</constant>
 		<constant name="STRETCH_KEEP_ASPECT" value="5">
+			Scale the texture to fit the node's bounding rectangle, but maintain the texture's aspect ratio.
 		</constant>
 		<constant name="STRETCH_KEEP_ASPECT_CENTERED" value="6">
+			Scale the texture to fit the node's bounding rectangle, center it and maintain its aspect ratio.
 		</constant>
 		<constant name="STRETCH_KEEP_ASPECT_COVERED" value="7">
```
Comment (Noshyaar) [inline doc/classes/TextureRect.xml]: Only one is missing its description. 😱

File: doc/classes/NinePatchRect.xml [hunk 3]
```diff
@@ -100,36 +102,49 @@
 	</methods>
 	<members>
 		<member name="axis_stretch_horizontal" type="int" setter="set_h_axis_stretch_mode" getter="get_h_axis_stretch_mode" enum="NinePatchRect.AxisStretchMode">
+      Doesn't do anything at the time of writing.
 		</member>
 		<member name="axis_stretch_vertical" type="int" setter="set_v_axis_stretch_mode" getter="get_v_axis_stretch_mode" enum="NinePatchRect.AxisStretchMode">
+      Doesn't do anything at the time of writing.
 		</member>
 		<member name="draw_center" type="bool" setter="set_draw_center" getter="is_draw_center_enabled">
+      If true, draw the panel's center. Else, only draw the 9-slice's borders.
```
Comment (Noshyaar) [inline doc/classes/NinePatchRect.xml]: `[code]` and default value?

File: doc/classes/TextureRect.xml [hunk 4]
```diff
@@ -56,29 +56,36 @@
 	</methods>
 	<members>
 		<member name="expand" type="bool" setter="set_expand" getter="has_expand">
-			If [code]true[/code] texture will expand to fit. Default value: [code]false[/code].
+			If [code]true[/code], the texture scales to fit its bounding rectangle. Default: [code]false[/code].
```
Comment (Noshyaar) [inline doc/classes/TextureRect.xml]: >Default:

If 'value' should be removed, please update the guideline.

Comment (Noshyaar) [reply doc/classes/TextureRect.xml]: https://github.com/GDquest/godot-docs/blame/24003933919f06c671e27aa01b6f93eea55b91d0/community/contributing/docs_writing_guidelines.rst#L317

Comment (NathanLovato) [reply doc/classes/TextureRect.xml]: There's no guideline for "Default value", only an example I pulled from the existing docs. Is the default value worth mentioning at all in every case?

Comment (Noshyaar) [reply doc/classes/TextureRect.xml]: IMO default value is quite useful with obscure discreet value like enum and boolean. Not so much for continuous value. I'm just not sure if I should use `Default value:` or `Default:`.

**Comment by NathanLovato** [comment]
Fixed all the issues. Did a fixup to merge the commits together.

File: doc/classes/NinePatchRect.xml [hunk 5]
```diff
@@ -100,36 +102,49 @@
...
+			Doesn't do anything at the time of writing.
 		</member>
 		<member name="axis_stretch_vertical" type="int" setter="set_v_axis_stretch_mode" getter="get_v_axis_stretch_mode" enum="NinePatchRect.AxisStretchMode">
+			Doesn't do anything at the time of writing.
 		</member>
 		<member name="draw_center" type="bool" setter="set_draw_center" getter="is_draw_center_enabled">
+			If [code]true[/code], draw the panel's center. Else, only draw the 9-slice's borders. Default value: [code]true[/code]
 		</member>
 		<member name="patch_margin_bottom" type="int" setter="set_patch_margin" getter="get_patch_margin">
+			The height of the 9-slice's bottom row. A margin of 16 means the 9-slice's bottom corners and side will have a height of 16 pixels. You can set all 4 margin values indidivually to create panels with non-uniform borders.
```
Comment (Noshyaar) [inline doc/classes/NinePatchRect.xml]: >indidivually

**Comment by NathanLovato** [comment]
Fixed!

**Pull request merged**